docs/guide/components/including-sub-components.md

```diff
@@ -225,3 +225,43 @@ In this case, when rendered the components will look like:
 You can see that in the `@search-box` component the button text is now the same as the text defined in the `button.config.yml` file.
 
 See the documentation on [referencing other component\'s data](../core-concepts/context-data.html#data-references) for more information on using this technique.
+
+### Fully rendering a sub-component
```

Comment:
can we add a "added in v1.3.0" tag here like we've used in [other places](https://fractal.build/guide/collections/configuration-reference.html#root)?

Reply:
@mihkeleidast done!